### DISCUSSION TITLE
Timeline for larger MIMIC-IV Waveform Database release

### DISCUSSION THREAD

**Discussion by finncatling** [MIT-LCP/mimic-code#1465] [MIMIC-IV]
Thanks very much for your work on MIMIC-IV Waveform Database v0.1.0.

I see from [this page](https://physionet.org/content/mimic4wdb/0.1.0/) that release of a larger waveform database is upcoming. I wondered if you had an estimate for when this release will occur, please?

Thanks again!

**Comment by schu** [comment]
I also would like to ask if you can give an update on the next release at this point?

> An upcoming release will include around 10,000 records.

Is it correct to assume that 10,000 records will be waveform data for roughly 20% of the ICU patients?

```
select count(distinct subject_id) from mimiciv_icu.icustays;

 count
-------
 50920
(1 row)
```
